### DISCUSSION TITLE
Help with implementation on new Dialog with monaco Editor

### DISCUSSION THREAD

**Discussion by eli88p** [eclipse-theia/theia#10595] [Q&A]
Hi guys, I need to help with injection on monaco editor to my dialog,
maybe someone have a example?
Thanks in advance
Eli.

**Comment by vince-fugnitto** [comment]
@eli88p thank you for the discussion, I believe you can use/inject [`MonacoEditorProvider`](https://eclipse-theia.github.io/theia/docs/next/classes/monaco.monacoeditorprovider.html) for you to embed editors in your dialog.

We make use of it for the console, and debug widget currently:

- https://github.com/eclipse-theia/theia/search?q=MonacoEditorProvider
